pcsx2/GS/GSLzma.h

```diff
@@ -1,4 +1,4 @@
-// SPDX-FileCopyrightText: 2002-2025 PCSX2 Dev Team
+	// SPDX-FileCopyrightText: 2002-2025 PCSX2 Dev Team
```

Comment:
Looks like there was a little accident here.

Apart from that this new code looks much better, although I haven't tested it. I always hated the std::enable_if stuff.